en/cheat-sheet.md

```diff
@@ -493,7 +493,7 @@ For example:
 {{< copyable "shell-regular" >}}
 
 ```shell
-helm inspect values pingcap/tidb-operator --version=v1.4.0-beta.3 > values-tidb-operator.yaml
```

Comment:
dup with https://github.com/pingcap/docs-tidb-operator/pull/2147 ?

Reply:
no, this is for the `master` branch. #2146 is for the `release-1.4` branch.